vitest.ts

```diff
@@ -1,7 +0,0 @@
-import createFetchMock from 'vitest-fetch-mock';
```

Comment:
Why are we removing this?

Reply:
Seems to not be used anywhere, and it was crashing with vitest 0.33. Everything runs fine without it, do you know where it is used ?